bpf/kmesh/cgroup_sock.c

```diff
@@ -116,20 +185,49 @@ static inline bool conn_from_cni_sim_delete(struct bpf_sock_addr *ctx)
     return ((bpf_ntohl(ctx->user_ip4) == 1) && (bpf_ntohl(ctx->user_port) == 0x3a20000));
 }
 
-SEC("cgroup/connect4")
-int cgroup_connect4_prog(struct bpf_sock_addr *ctx)
+static inline bool check_kmesh_managed_process(struct bpf_sock_addr *ctx)
 {
     if (conn_from_cni_sim_add(ctx)) {
-        record_netns_cookie(ctx);
+        record_kmesh_netns_cookie(ctx);
         // return failed, cni sim connect 0.0.0.1:929(0x3a1)
         // A normal program will not connect to this IP address
-        return CGROUP_SOCK_OK;
+        return true;
     }
+
     if (conn_from_cni_sim_delete(ctx)) {
-        remove_netns_cookie(ctx);
+        remove_kmesh_netns_cookie(ctx);
+        return true;
+    }
+    return false;
+}
+
+static inline bool check_bypass_process(struct bpf_sock_addr *ctx)
+{
+    if (conn_from_bypass_sim_add(ctx)) {
+        record_bypass_netns_cookie(ctx);
+        // return failed, cni sim connect 0.0.0.1:929(0x3a1)
+        // A normal program will not connect to this IP address
+        return true;
+    }
+    if (conn_from_bypass_sim_delete(ctx)) {
+        remove_bypass_netns_cookie(ctx);
+        return true;
+    }
+    return false;
+}
+SEC("cgroup/connect4")
+int cgroup_connect4_prog(struct bpf_sock_addr *ctx)
+{
+    if (check_kmesh_managed_process(ctx) || !check_kmesh_enabled(ctx)) {
```

Comment:
check_kmesh_managed_process and check_kmesh_enabled are very confusing names

Reply:
> check_kmesh_managed_process and check_kmesh_enabled are very confusing names

done,I have modify to handle_kmesh_manage_process and is_kmesh_enabled